### DISCUSSION TITLE
Chaining Athens

### DISCUSSION THREAD

**Discussion by ach5948** [gomods/athens#1706] [Q&A]
My use case might be a super edge case, but is there currently any way to have athens download packages from another proxy instead of directly from the version control?

My edge case is partially mirroring another Athens server in an offline environment. So I'd like to setup the server to do an async_redirect to the primary Athens server then download the relevant packages from the primary Athens server.

**Comment by arschles** [comment]
Hi @ach5948 would the [download mode file](https://docs.gomods.io/configuration/download/) work for that?

**Reply by ach5948** [reply]
Hey, from what I can tell, `downloadURL` in the download mode file is only used by `async_redirect` to direct users where to download when the proxy doesn't have the package, but the proxy then still tries to download from the Version Control Source rather than from `downloadURL`.
